lib/dotcom/map_helpers.ex

```diff
@@ -55,14 +55,14 @@ defmodule Dotcom.MapHelpers do
 
   @spec image(atom) :: String.t()
   def image(:subway) do
-    static_url(DotcomWeb.Endpoint, "/subway-map-image")
+    "/subway-map-image"
```

Comment:
I didn't use the `~p` sigils because those come in through `DotcomWeb`'s `view_helpers` macro, which generated conflicting function names with something else that's imported here.

I figured it wasn't a huge deal because `~p` sigils' main benefit is to let you know if you're using a path that isn't defined in `Router`, but we [match on all paths anyway](https://github.com/mbta/dotcom/blob/c892bd12d44cab84b7696d58729e7c64f5015cff/lib/dotcom_web/router.ex#L306).